system/datetime-plugin/plugin/src/__tests__/e2e.spec.ts

```diff
@@ -18,14 +18,15 @@ describe("e2e", () => {
     });
   });
 
-  test("currentTimestamp", async () => {
-    const result = await client.invoke<string>({
+  it("retrieves current datetime", async () => {
+    const result = await client.invoke({
       uri,
-      method: "currentTimestamp"
```

Comment:
currentTimestamp better describes the method than time because it's actually unix timestamp

Reply:
fixed. I updated the version from demos, not the one in integrations, and that's why this was changed.